packages/fiori/src/NotificationList.ts

```diff
@@ -128,7 +128,7 @@ class NotificationList extends UI5Element {
 	}
 
 	get innerList() {
-		return this.shadowRoot?.querySelector("ui5-notification-list-internal") as NotificationListInternal;
```

Comment:
Why do we need this?

Reply:
Yes, for now, otherwise I cannot push it